app/assets/stylesheets/layouts/_devise.pcss

```diff
@@ -0,0 +1,42 @@
+.layout-devise {
+  body {
+    @apply h-screen bg-gray-900;
+
+    background: {
+      image: url('../images/background.png');
+      repeat: no-repeat;
+      position: center;
+      size: cover;
+    }
+  }
+
+  .app-main {
+    @apply h-full w-full grid;
+
+    place-items: center;
+    background: linear-gradient(180deg, var(--color-white-200) -50%, var(--color-original-black) 100%);
+    mix-blend-mode: normal;
+    backdrop-filter: blur(40px);
+
+    &__container {
+      @apply max-w-sm mx-8;
+
+      @screen md {
+        @apply max-w-xs mx-auto;
+      }
+    }
+
+    &__logo {
+      @apply h-10;
+    }
+
+    &__description {
+      @apply h-6 mt-6;
+      @apply text-center text-white-600 text-base font-normal tracking-normal;
```

Comment:
Is `tracking-normal` really needed since `letter-spacing: 0` should be the default value no?